modules/mono/glue/GodotSharp/GodotSharp/Core/Color.cs

```diff
@@ -987,32 +1002,49 @@ public override bool Equals(object obj)
             return false;
         }
 
+        /// <summary>
+        /// Returns <see langword="true"/> if this color and <paramref name="other"/> are equal
+        /// </summary>
+        /// <param name="other">The other <see cref="Color"/> to compare.</param>
+        /// <returns>Whether or not the <see cref="Color"/>s are equal.</returns>
```

Comment:
I'm fine with either, my personal preference would be to use it whenever possible since an IDE would be able to rename it everywhere when refactoring as well as link to it, but I guess it's unnecessary in this case.

Reply:
That being said, the benefits of these docs outweigh this, so it doesn't prevent merging the PR.

Reply:
It seems in the .NET docs that the types are indeed used to complement other nouns rather than being nouns by themselves. This does seem a bit strange though. I think I would prefer `this <see cref="Color"/>` to `this color` in the summary even if `this <see cref="Color"/> instance` or similar is better.

Reply:
Here's a case where they do use the type as a noun: https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.count?view=net-5.0